### PR TITLE
LPS-162459,LPS-162456

### PR DESCRIPTION
**Pull request by rodrigopaulino**

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
Hi, @carolmariaabb. Just want to let you know that you've already reviewed the commits related to LPS-162456 [here](https://github.com/liferay-objects/liferay-portal/pull/1047). You just need to review the last one. 🙂 
Thx.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java [hunk 1]
```diff
@@ -117,6 +134,50 @@ private void _executeObjectActions(
...
+	private List<Attribute> _getModifiedAttributes(
+		ObjectEntry originalObjectEntry, ObjectEntry objectEntry) {
+
+		List<Attribute> attributes = new ArrayList<>();
+
+		Map<String, Serializable> originalValues =
+			originalObjectEntry.getValues();
+		Map<String, Serializable> values = objectEntry.getValues();
+
+		for (Map.Entry<String, Serializable> entry :
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java:164]: @rodrigopaulino I would do
`_getModifiedAttributes(Map<String, Serializable> originalValues,  objectEntry.getValues())` just to buy two lines 😆 

But there's no need to change this.

**Review by carolmariaabb — APPROVED** [review]
LGTM 🥳

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java:164]: Oh right! Nice catch. I'll wait for the results (which will most certainly fail) and change this. Thx, @carolmariaabb

**Comment by rodrigopaulino** [comment]
ci:stop

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java]: Done

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by rodrigopaulino** [comment]
ci:reevaluate:1267401_5046

**Comment by carolmariaabb** [comment]
ci:reevaluate:1267401_5046

**Comment by carolmariaabb** [comment]
ci:test:relevan

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
@carolmariaabb We have some conflicts in this branch :/

**Comment by rodrigopaulino** [comment]
ci:close

**Comment by rodrigopaulino** [comment]
I'm sending a new one shortly

**Comment by rodrigopaulino** [comment]
Resent https://github.com/liferay-objects/liferay-portal/pull/1103

**Pull request closed without merging**